cmd/lakectl/cmd/doctor.go

```diff
@@ -11,11 +11,42 @@ import (
 	"github.com/treeverse/lakefs/pkg/api"
 )
 
-var (
-	ErrCredential       = errors.New("credential error")
-	ErrWrongEndpointURI = errors.New("wrong endpoint_uri error")
-	ErrUnknownConfig    = errors.New("unknown configuratioin error")
-)
+type ErrCredential struct {
+	Message string
+	Details string
+}
+
+func (e *ErrCredential) Error() string {
+	return (e.Message + "\n" + e.Details + "\n")
+}
+
```

Comment:
For future reference (none of this is required in this PR, but it would be more Go-ish if you did do these): these errors work, but are a bit dirty.  They break the errors wrapping chain.  You could have instead

```go
type WrappingError struct {
	Message string
	DetailedErr error
}

func (e *WrappingError) Error() string { return e.Message }

func (e *WrappingError) Unwrap() error { return e.DetailedErr }

func (e *WrappingError) Details() string { return c.DetailedErr.Error() }
```

The `Unwrap` call means the `errors.Is` and `errors.As` will continue to work on this error.

Now copy this type into every desired type, e.g.
```go
type WrongEndpointURIError WrappingError

var ErrWrongEndpointURI = WrongEndpointERIError{}
```

and you get all the parts in one place.